sql/core/src/test/resources/sql-tests/inputs/double-quoted-identifiers.sql

```diff
@@ -0,0 +1,107 @@
+-- test cases for spark.sql.ansi.double_quoted_identifiers
+
+-- Base line
+SET spark.sql.ansi.double_quoted_identifiers = false;
```

Comment:
Why we need to test `false`? I assume all other existing test are testing double quoted identifiers?

Though maybe a bit more tests are better.

Reply:
I am thinking if we want to both cases, we can split it into two files and one is the config off and another is the config on. For the config on case, that file can be put under `inputs/ansi/`

Reply:
The only QA I could find for double-quoted STRINGs (false) were low level unit tests. So I think it is reasonable to have a SQL level baseline.
The total number of tests isn't really big enough to justify the overhead of running it in multiple modes with distinct golden files.
Keeping it in one file with one golden files to me looks more maintainable.

Reply:
I think it is fine to have all cases in one file, considering the number of test cases is small.

Reply:
It's small today, but when we add more tests later, let's split.